src/components/AgmLocationChange/AgmYear.vue

```diff
@@ -40,12 +39,6 @@ export default class AgmYear extends Vue {
     this.emitValid(this.value)
   }
 
-  @Watch('value')
-  @Debounce(300)
-  onValueChanged (val: string): void {
```

Comment:
Not needed. This way we'll also stay consistent between all the other filings.

Reply:
Does this component emit the validity flag on every character typed? The purpose of the debounce was to reduce the "noise" coming from this component.

Reply:
You're absolutely right. Returned the debounce + fixed tests accordingly.